.cirrus.yml

```diff
@@ -312,16 +312,16 @@ task:
     FILE_ENV: "./ci/test/00_setup_env_mac.sh"
 
 task:
-  name: 'macOS 12 native x86_64 [gui, system sqlite] [no depends]'
+  name: 'macOS 13 native arm64 [gui] [no depends]'
```

Comment:
Is this no longer using sqlite? I think we should have one task that runs sqlite-only

Reply:
Using system `sqlite` package is by default on macOS since #25985.

Reply:
Well, it should mention that this is sqlite-only (no bdb). But it looks like you removed the "only" in 0bb7a1f71db1d986ab824d114534fb7671024990 ?

Reply:
> Well, it should mention that this is sqlite-only (no bdb).

Maybe install `berkeley-db@4` package?

Reply:
No, the goal is to *not* have bdb, so that sqlite-only (test) bugs are detected

Reply:
Updated.